nixos/modules/services/mail/stalwart-mail.nix

```diff
@@ -62,6 +64,9 @@ in {
       resolver.public-suffix = lib.mkDefault [
         "file://${pkgs.publicsuffix-list}/share/publicsuffix/public_suffix_list.dat"
       ];
+      config.resource = {
+        spam-filter = lib.mkIf cfg.spam-filter "file://${cfg.package}/etc/stalwart/spamfilter.toml";
+      };
```

Comment:
That was actually the first logic I had also, but I wasn't sure how to use the imperative then.

I wasn't able test yet what happens when I see the value to "false"., doesn't it treat as URL later and will print and error?

If not could you point me to the code that is treating the "false"?
I'll try to test that later on my machine.

Anyways we could still disable it with overriding to `null` which is possibly also just okay for me.

Reply:
I didn't meant to disable spam filter, but use the downloaded.
And my doubt is that setting that to something like false will raise an error (and downloading it after that anyways), but I like to prevent a possible error in the log.